geode-docs/tools_modules/gfsh/command-pages/create.html.md.erb

```diff
@@ -504,6 +504,7 @@ create gateway-sender --id=value --remote-distributed-system-id=value
    [--maximum-queue-memory=value] [--alert-threshold=value] [--dispatcher-threads=value] 
    [--order-policy=value][--gateway-event-filter=value(,value)*] 
    [--gateway-transport-filter=value(,value)*]
+   [--group-transaction-events=value]
```

Comment:
I think this should be `--group-transaction-events(=value)?` , since the value has a specified default value of true and so doesn't need to be included in all cases.

Reply:
You are right. The same should apply to --parallel. Right?

Reply:
You are right.
It should also be so in --parallel so I have corrected the documentation there too.